test/hotspot/jtreg/runtime/interpreter/CountBytecodesTest.java

```diff
@@ -0,0 +1,77 @@
+/*
+ * Copyright (c) 2025, Oracle and/or its affiliates. All rights reserved.
+ * Copyright (c) 2025 SAP SE. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+
+/*
+ * @test
+ * @bug 8350642
+ * @requires vm.debug & vm.bits == "64"
+ * @summary Test the output for CountBytecodes and validate that the counter
+ *          does not overflow for more than 2^32 bytecodes counted.
+ * @library /test/lib
+ * @run main/othervm/timeout=300 CountBytecodesTest
```

Comment:
The long tests should be excluded from tier1. Please update TEST.groups.

Reply:
I excluded the test from the `tier1_runtime` tests. To my understanding it should now run in tier4. Could you please verify? Thanks.

Reply:
Yes, this should work. In current definition, `tier4` is "catch-all" group that handles all the tests that are not explicitly in `tier{1,2,3}`.